pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -100,6 +100,10 @@ func (mixerplugin) OnOutboundListener(in *plugin.InputParams, mutable *plugin.Mu
 		"context.reporter.kind": attrStringValue("outbound"),
 	}
 
+	if network, found := in.Node.Metadata[model.NodeMetadataNetwork]; found {
+		attrs["context.reporter.network"] = attrStringValue(network)
```

Comment:
You need to define this in the manifest https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml#L103 in order to write expressions on it.

Reply:
@kyessenov thanks for pointing that out